main.go

```diff
@@ -127,6 +128,7 @@ func main() {
 	flag.StringVar(&outputPath, "log-file", "stderr", "The path to redirect controller logs")
 	flag.StringVar(&introspectBindAddr, "introspect-bind-addr", ":22775",
 		"Port for serving the introspection API")
+	flag.BoolVar(&leaseOnly, "lease-only", false, "Controller uses lease only for leader election")
```

Comment:
Since this will applied to all EKS versions and we only want to make 1.25 and following versions using Lease only, making it default to false can avoid changing all older manifests in older versions, and just need to add the flag for >= 1.25. After all older versions deprecate, we can update the default value in code to true and let flags in manifests become inactive and deprecate natively with EKS updates.

Reply:
Got it, thanks for explaining!